documentation/api.md

```diff
@@ -504,6 +504,57 @@ access_token | None | (Required) Access Token of submitter
 #### Response
 See example
 
+## Comments
+>><h4> Example Request </h4>
+> ```python
+import requests
+data = {
+    'filename': 'lab00.py',
+    'line': 2,
+    'message': "Great job.\nConsider the use of the `total` variable."
+}
+url = "https://okpy.org/api/v3/backups/a24x23/comment?access_token={}"
+access_token = 'test'
+r = requests.post(url.format(access_token), data=data)
+response = r.json()
+```
+>><h4> Response </h4>
+```
+{
+    "data": {
+        'html': "<div class='comment'> ... </div>"
```

Comment:
The HTML is pretty specific to our frontend (e.g. it includes "edit" and "close" buttons), so it shouldn't be included in our API response. The client can just use their own markdown renderer if they want a preview.